numpy/lib/tests/test_arraysetops.py

```diff
@@ -34,7 +34,25 @@ def test_intersect1d(self):
         assert_array_equal(c, ed)
 
         assert_array_equal([], intersect1d([], []))
-
+    def test_intersect1d_indices(self):
```

Comment:
PEP8 needs a newline above this line, and below the last line of the function

Reply:
okay i added one after the function as well (same problem)